benches/benchmarks.rs

```diff
@@ -162,11 +163,40 @@ fn decoding_benchmarks(c: &mut Criterion) {
     group.finish();
 }
 
+fn generating_random_bytes(c: &mut Criterion) {
+    let mut group = c.benchmark_group("Generating random bytes");
+
+    for n in [10, 100, 1_000, 10_000, 100_000, 1_000_000] {
```

Comment:
With the whole overhead if our benchmark would be invoking new `nu` processes for each sample, I don't think there is much to glance from the low numbers. 

In general I am not sure how relevant it is to have a small scaling factor. Criterion will run the benchmark often enough to get stable data, so each scaling step will take a not insignificant time overall.